File: Breeze.Domain/Budget.cs
namespace Breeze.Domain
{
    public class Budget
    {
        public int Id { get; set; }
        public string UserId { get; set; }
        public decimal MonthlyIncome { get; set; }
        public decimal MonthlyExpenses { get; set; }
        public DateOnly Date { get; set; }
    }
}

File: Breeze.Web/app/budget/types/budget.ts
import z from 'zod';

export interface Budget {
  id: string;
  userId: string;
  monthlyIncome: string;
  monthlyExpenses: string;
  date: string;
  createdAt: string;
  updatedAt: string;
}

export const budgetFormSchema = z.object({
  monthlyIncome: z.string().min(1, 'Monthly income is required'),
  monthlyExpenses: z.string().min(1, 'Monthly expenses is required'),
});

export interface BudgetFormData {
  monthlyIncome: string;
  monthlyExpenses: string;
}
